actionpack/lib/action_dispatch/http/request.rb

```diff
@@ -199,6 +199,23 @@ def headers
       @headers ||= Http::Headers.new(self)
     end
 
+    # Early Hints is an H2 status code that indicates hints to help a client start
+    # making preparations for processing the final response.
+    #
+    # If the env contains +rack.early_hints+ then the server accepts HTTP2 push for Link headers.
+    #
+    # The +send_early_hints+ method accepts an array of links as follows:
+    #
+    #   send_early_hints({ link: "path/to/stylesheet.css", as: "stylesheet}, { link: "path/to/javascript.js", as: "script"})
```

Comment:
Missing a closing quote on `as: "stylesheet`. Should add a space before the two `}` as well.